### PR TITLE
ontoavida - freeform markdown: html and padding styles added

### PR DESCRIPTION
**Pull request by raul-ortega**
Regarding [PR #2044](https://github.com/OBOFoundry/OBOFoundry.github.io/pull/2044) and [issue #2047](https://github.com/OBOFoundry/OBOFoundry.github.io/issues/2047), I've added a extra html tag and padding to it and the image to solve it.

### CODE REVIEW

**Comment by cthoyt** [comment]
Again, there's already a fix for the styling on this page in #2042 that should make the styling better by default. Please hold tight until that one is merged

**Comment by matentzn** [comment]
@cthoyt can you advice what I should do with this PR?

**Comment by cthoyt** [comment]
Remove the unnecessary styling then make a new screenshot and if looks good merge it

**Comment by matentzn** [comment]
Thanks @cthoyt. @raul-ortega sounds good?

File: ontology/ontoavida.md [hunk 1]
```diff
@@ -30,4 +30,5 @@ tags:
 tracker: https://gitlab.com/fortunalab/ontoavida/-/issues
 activity_status: active
 ---
-<img  src="https://fortunalab.org/images/alife_bacteria.jpg" height="150px" align="left"/>The **Ontology for Avida ([OntoAvida](https://gitlab.com/fortunalab/ontoavida))** project aims to develop an integrated vocabulary for the description of [Avida](https://github.com/devosoft/avida), the most widely used computational approach for performing experimental evolution. The lack of a clearly defined vocabulary makes biologists feel reluctant to embrace the field of digital evolution. This unique ontology has the potential to change this picture overnight. In addition, OntoAvida will allow researchers to make inference (e.g., on phenotypic plasticity) based on certain rules and constraints, facilitate the reproducibility of the *in silico* evolution experiments reported in the [scientific literature](https://gitlab.com/fortunalab/ontoavida#references), and trace the provenance of the data stored in AvidaDB, a semantic database that stores genomes and transcriptomes of more than a million digital organisms.
+<p style="padding-top: 10px; text-align:justify;"/>
```
Comment (cthoyt) [inline ontology/ontoavida.md:33]:
```suggestion
```

**Comment by raul-ortega** [comment]
Due to the removal of the p tag there is no space between the tab buttons and the customized content.

**Comment by cthoyt** [comment]
We need to go one more round to update the ontology summary page, you shouldn't be responsible for assigning style information in a single ontology

**Pull request merged**